src/Type/FileTypeMapper.php

```diff
@@ -232,10 +233,12 @@ function (\PhpParser\Node $node) use ($fileName, $lookForTrait, &$phpDocMap, &$c
 						$uses[strtolower($use->getAlias()->name)] = sprintf('%s\\%s', $prefix, (string) $use->name);
 					}
 					return null;
+				} elseif ($node instanceof Node\Stmt\ClassMethod) {
+					$functionName = ltrim(sprintf('%s\\%s', $namespace, $node->name->name), '\\');
```

Comment:
This doesn't seem right. Let's say we have class `Bar` in namespace `Foo` with method `baz`. This will produce `Foo\baz`...

Reply:
Thanks !